src/scenario.h

```diff
@@ -302,7 +302,7 @@ typedef struct {
 	char custom_strings[0x8000];
 	uint32 game_ticks_1;
 	rct_ride rides[255];
-	uint16 word_01388698;
+	uint16 word_01388698; // TODO: rename to match RCT2_ADDRESS_SAVED_AGE ???
```

Comment:
I think the problem lies in `rideIndex` getting to 255. Whereas 254 is the last possible, before it clashes with other addresses. I very well might be wrong though.

```
0x013628F8             + ((0x208 + 0x58)    * 255) = 0x1388698
RCT2_ADDRESS_RIDE_LIST + (size of rct_ride  * 255) = RCT2_ADDRESS_SAVED_AGE
```